DistroLauncher/DistributionInfo.h

```diff
@@ -8,13 +8,19 @@
 namespace DistributionInfo
 {
     // The name of the distribution. This will be displayed to the user via
-    // wslconfig.exe and in other places. This value must not change between
-    // versions, otherwise users upgrading from older versions will see launch
-    // failures.
+    // wslconfig.exe and in other places. It must conform to the following
+    // regular expression: ^[a-zA-Z0-9._-]+$
     //
-    // It must only conform to the following regular expression: ^[a-zA-Z0-9._-]+$
+    // WARNING: This value must not change between versions of your app,
+    // otherwise users upgrading from older versions will see launch failures.
     const std::wstring Name = L"MyDistribution";
 
-    // The title bar for the console window while the distrubiton is installing.
+    // The title bar for the console window while the distribution is installing.
     const std::wstring WindowTitle = L"My Distribution";
+
+    // Create and configure a user account.
+    bool CreateUser(const std::wstring& userName);
+
+    // Generate a command to query the UID of the user account.
+    std::wstring QueryUidCommand(const std::wstring& userName);
```

Comment:
worried about the impedance mismatch here: `CreateUser` generates _and runs_ the command to create the user... where `QueryUidCommand` creates the command for somebody else to run.

Reply:
Good call, probably better to have it called QueryUid and return the UID.